addons/web/__manifest__.py

```diff
@@ -196,14 +196,38 @@
 
             ('include', 'web._assets_bootstrap'),
 
+            'web/static/src/env.js',
```

Comment:
haha, it looks like you are getting to what I predicted: 1 bundle for frontend and 1 bundle for backend

Reply:
I'm trying this approach because you suggested it, self fulfilling prophecy?

Reply:
Actually you warned me about this but reading this conversation I wonder why not in common? :thinking: I guess too much "common" places which don't need that? Not sure, is not it worth to at least add those in a common template which is 'include' in both assets?

Reply:
@ged-odoo suggested that we stop having a common bundle altogether to make asset management easier: in web we need all of core so we can just use one glob (`/wev/static/src/core/**/*.js`) which makes it easy to see at a glance that we use everything, whereas in frontend we really want to pick and choose only what we need to reduce the bundle size. I think the end goal was to completely avoid having a "common" bundle and simply have one bundle per use case, Géry might be able to tell you more about the exact reasoning. I'm open to moving most of this to common though, since most of this is in fact common and needed to have the basics working (error service, RPC, notificaitons)

Reply:
Same advice on the **assets** point of view (like there is no real value of having a common bundle including stuff that is loaded separately and that we are never too sure we need entirely). But picking stuff in core seems unperfect IMHO = you have to know what core module need each other and, also, what is in "core" should be available everywhere even if unused IMHO (if it is something very specific, it should not be in core).

In the end: maybe just a matter of splitting the framework "core" in "core-for-backend" and "core-for-odoo"? (not with those names obviously)